.hyperagent/census_metrics.py

```
@@ -0,0 +1,193 @@
+    grand = n_items * n_raters
+    p_cat = [sum(row[j] for row in table) / grand for j in range(n_categories)]
+    p_exp = sum(p * p for p in p_cat)
+    if math.isclose(p_exp, 1.0):
+        return 1.0  # perfect expected agreement: kappa undefined; report 1.0
+    return (p_bar - p_exp) / (1 - p_exp)
+
+
+# --- set agreement: Jaccard and MASI (Passonneau 2006) ---------------------
+
+def jaccard(a: set, b: set) -> float:
+    """|A ∩ B| / |A ∪ B| (intersection over union). Two empty sets count as identical (1.0)."""
```

Ruff (0.15.18)

[warning] Docstring contains ambiguous `∪` (UNION). Did you mean `U` (LATIN CAPITAL LETTER U)?

(RUF002)